lib/matplotlib/widgets.py

```diff
@@ -2325,12 +2333,12 @@ def onselect(eclick: MouseEvent, erelease: MouseEvent)
         else:
             props = dict(markeredgecolor=rectprops.get('edgecolor', 'r'))
         props.update(cbook.normalize_kwargs(marker_props, Line2D._alias_map))
-        self._corner_order = ['NW', 'NE', 'SE', 'SW']
+        self._corner_order = ['SW', 'SE', 'NE', 'NW']
         xc, yc = self.corners
         self._corner_handles = ToolHandles(self.ax, xc, yc, marker_props=props,
                                            useblit=self.useblit)
 
-        self._edge_order = ['W', 'N', 'E', 'S']
+        self._edge_order = ['W', 'S', 'E', 'N']
```

Comment:
This order hurts my brain ;-)  Cardinal directions should be clockwise and start w/ north!.  But, I don't think it really matters here....